ast/astutil/format_test.go

```diff
@@ -279,11 +279,9 @@ func TestFormatWithComments(t *testing.T) {
 
 	if want := `// hi
 // there
-{
```

Comment:
@Marwes @wolffcm This is the file where I made a change the the expected output. Its a little bit weird, but I think that the final output makes sense for the most part.

Reply:
I'd probably expect that the line comment forced the record onto multiple lines, but the formatting before didn't indent the comment so it (arguably) also weren't consistent.